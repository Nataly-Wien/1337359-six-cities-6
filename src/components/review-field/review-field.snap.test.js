import React from 'react';
import {render} from '@testing-library/react';
import ReviewField from './review-field';

it(`Should review textarea render correctly`, () => {
  const onChange = () => { };
  const review = `Lovely bright space and comfortable beds. We had two apartments, one at the front and one at the back. The apartment at the back was lovely and roomy, the one at the front pretty cramped, but good value for money.`;

  const {container} = render(<ReviewField onChange={onChange} review={review} />);

  expect(container).toMatchSnapshot();
});
