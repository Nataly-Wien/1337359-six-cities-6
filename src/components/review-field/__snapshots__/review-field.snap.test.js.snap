// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should review textarea render correctly 1`] = `
<div>
  <textarea
    class="reviews__textarea form__textarea"
    data-testid="review"
    id="review"
    name="review"
    placeholder="Tell how was your stay, what you like and what can be improved"
  >
    Lovely bright space and comfortable beds. We had two apartments, one at the front and one at the back. The apartment at the back was lovely and roomy, the one at the front pretty cramped, but good value for money.
  </textarea>
</div>
`;
